### ISSUE TITLE
Tutorial Site Jumps around when typing

### ISSUE THREAD

**Issue by JacobGoldenArt** [sveltejs/svelte.dev#582]
### Describe the bug

As I went through the tutorial for the new Svelte 5 tutorial, there are certain lessons that when you are typing in the editor, the line you are typing on jumps out of view while you cursor still remains on the line you are typing on. My intuition is that as certain things are typed (perhaps state related code) the preview window is reactively reloading (which is the expected behavior) but I think this might be causing the editor to jump around, which makes you have to either finish typing you code blindly or you have to continuously scroll the editor back to the line you are typing on. btw...The new tutorial (content wise) is fantastic!

### Reproduction

https://svelte.dev/tutorial/svelte/actions

https://github.com/user-attachments/assets/841f1ef1-234b-43ab-a909-680749103a3d

### System Info

```shell
browsers Tested: 
Arc: Version 1.63.1 (54714) Chromium Engine Version 129.0.6668.101 Mac OS
Chrome: Version 129.0.6668.103 (Official Build) (arm64) : Mac Os
Safari: Version 18.0 (20619.1.26.31.6)
```

### Severity

annoyance

**Comment by JacobGoldenArt** [comment]
Thanks for working on this all! I'm getting through the advanced tutorial now but it is a test in patience! : )

**Comment by brunnerh** [comment]
Essentially duplicate of:
- #483

(I referenced the playground instead of the tutorial section)

**Comment by Rich-Harris** [comment]
closed via #597
